internal/ansible/watches/watches_test.go

```diff
@@ -102,7 +102,7 @@ func TestNew(t *testing.T) {
 	}
 }
 
-func TestLoad(t *testing.T) {
+func TestLoad(t *testing.T) { //nolint:gocyclo
```

Comment:
This one should be migrated to use the libs adopted for the tests so it might be solved in the future.